app/Providers/AppServiceProvider.php

```diff
@@ -33,17 +34,19 @@ public function boot(): void
         $tags = Tag::all();
         $features = Feature::all();
         $maxPrice = Hotel::getMaxPrice();
+        $titles = TopicCategory::all();
```

Comment:
rename to $topicCategories ($titles is too generic and not clear) and you are fetching topic categories here, not their titles